src/main/java/Game.java

```diff
@@ -19,7 +19,7 @@
 
     Game() {
         GameFacade gameFacade = new GameFacade();
-        gameFacade.createGame(new Rectangle(new Vector(0, 0), new Vector(100, 100)));
+        gameFacade.setupGameElements(new Rectangle(new Vector(0, 0), new Vector(100, 100)));
```

Comment:
1) pass gameField using constructor
2) refactor rectangle to GameField extends GameObject.